sysbuild/Kconfig.suit

```diff
@@ -92,15 +92,28 @@ config SUIT_RECOVERY_APPLICATION_DEFAULT
 
 endchoice
 
+choice SUIT_RECOVERY_APPLICATION_IMAGE_MANIFEST
+	prompt "Select which manifest controls the recovery application FW image"
+	default SUIT_RECOVERY_APPLICATION_IMAGE_MANIFEST_APP_LOCAL_3
+
+config SUIT_RECOVERY_APPLICATION_IMAGE_MANIFEST_APP_LOCAL_3
+	bool "The recovery application FW image is controlled by APP_LOCAL_3 manifest"
+
+config SUIT_RECOVERY_APPLICATION_IMAGE_MANIFEST_APP_RECOVERY
+	bool "The recovery application FW image is controlled directly by APP_RECOVERY manifest"
+
+endchoice
+
 config SUIT_RECOVERY_APPLICATION_PATH
 	string
 	default "${ZEPHYR_NRF_MODULE_DIR}/samples/suit/recovery" if SUIT_RECOVERY_APPLICATION_DEFAULT
 	help
-	  Source directory of the SUIR recovery application.
+	  Source directory of the SUIT recovery application.
 
 config SUIT_ENVELOPE_APP_RECOVERY_TEMPLATE_FILENAME
 	string "Name of the default root envelope template"
-	default "app_recovery_envelope.yaml.jinja2"
+	default "app_recovery_envelope.yaml_app_local.jinja2" if SUIT_RECOVERY_APPLICATION_IMAGE_MANIFEST_APP_LOCAL_3
```

Comment:
This is a pretty strange name - could you move the `yaml` part next to the `jinja2` part?

Reply:
This was a mistake, fixed